LiteNetLib/Utils/NetDataReader.cs

```diff
@@ -9,7 +9,6 @@ public class NetDataReader
         protected byte[] _data;
         protected int _position;
         protected int _dataSize;
-        private int _offset;
```

Comment:
private fields - works faster than get/set - so this is also intentionally created